impl/src/main/java/io/kroxylicious/testing/kafka/invm/InVMKafkaCluster.java

```diff
@@ -134,7 +132,7 @@ private static void prepareLogDirsForKraft(String clusterId, KafkaConfig config,
                     boolean.class);
             // note ignoreFormatter=true so tolerate a log directory which is already formatted. this is
             // required to support start/stop.
-            formatCommand.invoke(null, LOGGING_PRINT_STREAM, directories, metaProperties, MINIMUM_BOOTSTRAP_VERSION, true);
+            formatCommand.invoke(null, LOGGING_PRINT_STREAM, directories, metaProperties, MetadataVersion.latestProduction(), true);
```

Comment:
This change is required for SCRAM support.  This change should be transparent for users as the anticipated use case for the extension is ephemeral clusters spun up for the lifetime of a junit test.